tests/cases/fourslash/codeFixInferFromUsageConstructor.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path='fourslash.ts' />
+// @strictNullChecks: true
+
+////class TokenType {
+////    label;
+////    token;
+////    constructor([|label, token? |]) {
```

Comment:
If the user forgot type annotations they would probably forget `?` -- I think this should work without it.

Reply:
Moved to #28377